homeassistant/components/sensor/co2signal.py

```diff
@@ -4,16 +4,21 @@
 For more details about this platform, please refer to the documentation
 """
 import logging
+from datetime import timedelta
 
 import voluptuous as vol
 
 import homeassistant.helpers.config_validation as cv
 from homeassistant.const import (
-    ATTR_ATTRIBUTION, CONF_TOKEN, CONF_LATITUDE, CONF_LONGITUDE)
+    ATTR_ATTRIBUTION, CONF_TOKEN, CONF_LATITUDE, CONF_LONGITUDE,
+    CONF_SCAN_INTERVAL)
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import track_time_interval
+from homeassistant.util import Throttle
 
 CONF_COUNTRY_CODE = "country_code"
+SCAN_INTERVAL = timedelta(seconds=60)
```

Comment:
This is the only thing that's needed to change the default scan interval. Keep this and remove the rest.